CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -0,0 +1,74 @@
+  * Add `antctl mc get|delete membertoken` commands to get/delete member token. ([#4254](https://github.com/antrea-io/antrea/pull/4254), [@bangqipropel])
+- Add rule name to Audit Logging for Antrea-native policies. ([#4178](https://github.com/antrea-io/antrea/pull/4178), [@qiyueyao])
+- Add Service health check similar to kube-proxy in antrea-agent; it provides HTTP endpoints "<nodeIP>:<healthCheckNodePort>/healthz" for querying number of local Endpoints of a Service. ([#4120](https://github.com/antrea-io/antrea/pull/4120), [@shettyg])
+- Add S3Uploader as a new exporter of Flow Aggregator, which periodically exports expired flow records to AWS S3 storage bucket. ([#4143](https://github.com/antrea-io/antrea/pull/4143), [@heanlan])
+- Add scripts and binaries needed for running Antrea on non-Kubernetes Nodes (ExternalNode) in release assets. ([#4266](https://github.com/antrea-io/antrea/pull/4266) [#4113](https://github.com/antrea-io/antrea/pull/4113), [@antoninbas] [@Anandkumar26])
+
+### Changed
+
+- AntreaProxy now supports more than 800 Endpoints for a Service. ([#4167](https://github.com/antrea-io/antrea/pull/4167), [@hongliangl])
+- Add OVS connection check to Agent's liveness probes for self-healing on OVS disconnection. ([#4126](https://github.com/antrea-io/antrea/pull/4126), [@tnqn])
+- antrea-agent startup scripts now perform cleanup automatically on non-Kubernetes Nodes (ExternalNode) upon Node restart. ([#4277](https://github.com/antrea-io/antrea/pull/4277), [@Anandkumar26])
+- Make tunnel csum option configurable and default to false which avoids double encapsulation checksum issues on some platforms. ([#4250](https://github.com/antrea-io/antrea/pull/4250), [@tnqn])
+- Use standard value type for k8s.v1.cni.cncf.io/networks annotation. ([#4146](https://github.com/antrea-io/antrea/pull/4146), [@antoninbas])
+- Update Go to v1.19. ([#4106](https://github.com/antrea-io/antrea/pull/4106), [@antoninbas])
+- Manifest files in the main branch switch to the docker.io registry for "latest" antrea images. ([#4235](https://github.com/antrea-io/antrea/pull/4235), [@antoninbas])
+- Add API support for reporting Antrea NetworkPolicy realization failure. ([#4248](https://github.com/antrea-io/antrea/pull/4248), [@wenyingd])
+- Update ResourceExport's json tag to lowerCamelCase. ([#4211](https://github.com/antrea-io/antrea/pull/4211), [@luolanzone])
+- Add clusterUUID column to S3 uploader and ClickHouseExporter to support Multi-cluster. ([#4214](https://github.com/antrea-io/antrea/pull/4214), [@heanlan])
+
+### Fixed
+
+- Fix nil pointer error when collecting support bundle from Agent fails . ([#4306](https://github.com/antrea-io/antrea/pull/4306), [@tnqn])
+- Set no-flood config for TrafficControl ports after restarting Agent to prevent packets loops. ([#4318](https://github.com/antrea-io/antrea/pull/4318), [@hongliangl])
```

Comment:
```suggestion
- Set no-flood config for TrafficControl ports after restarting Agent to prevent ARP packets loops. ([#4318](https://github.com/antrea-io/antrea/pull/4318), [@hongliangl])
```

Reply:
added